smartsim/_core/_cli/build.py

```diff
@@ -399,18 +421,22 @@ def check_tf_install(self) -> None:
                 "tensorflow", self.versions.TENSORFLOW
             ):
                 msg = (
-                    f"TensorFlow {self.versions.TENSORFLOW} not installed in Python environment. "
-                    + f"Consider installing tensorflow=={self.versions.TENSORFLOW} with pip"
+                    "TensorFlow %s not installed in Python environment. "
+                    + "Consider installing tensorflow==%s with pip",
```

Comment:
%s should be {}?

Reply:
delete +

Reply:
replaced borked string with version that didn't go over line-length limit:
```
tf_version = self.versions.TENSORFLOW
msg = (
    f"TensorFlow {tf_version} not installed in Python environment. "
    f"Consider installing tensorflow=={tf_version} with pip"
)
```